src/main.rs

```diff
@@ -596,25 +561,72 @@ fn get_setup(args: &[String]) -> Setup {
         "PORT",
     );
 
+    let args: Vec<_> = std::env::args_os()
+        .filter_map(|s| match s.into_string() {
+            Ok(valid) => Some(valid),
+            Err(s) => {
+                eprintln!(
+                    "Command line argument was not valid Unicode and will not be evaluated: {:?}",
```

Comment:
Do we really need this error message? If not you could just do `s.into_string().ok()` I think.

Reply:
`s.into_string().ok()` will cause it to be silently skipped. Not what we want. We want to let the user know that the input was not valid unicode.

Reply:
`"...{:?}", s` will print the closest approximation that Rust can muster to the input. Being that `s` in that case will be invalid unicode it will replace invalid unicode with placeholders.

Reply:
I can live with it but don't think that non-valid Unicode will occur very often.

Reply:
> I can live with it but don't think that non-valid Unicode will occur very often.

I don't think so either that's the most important reason to include the message. Otherwise people doing future debugging could end up scratching there heads because they didn't consider such a rare case.